tests/lib/lang/evaluators/order4/mod.spec.ts

```diff
@@ -27,6 +29,49 @@ describe( "evalModExpression", () => {
         ])
     })
 
+
+
+    test("2 mod 4 = 2", () => {
```

Comment:
If possible, please keep with naming conventions of tests by describing the property being tested, e.g. "computes the remainder of divisions with a quotient of zero"